application/views/invoice_templates/pdf/InvoicePlane - overdue.php

```diff
@@ -286,5 +286,11 @@
     <?php endif; ?>
 </footer>
 
+<htmlpagefooter name="footer">
```

Comment:
Is that a new html tag?

Reply:
i have to look if this is a mpdf specific thing or we could use footer instead. the current footer part in the template is probably only text inserted at the end of the invoice and not a "real" footer shown on all pages?

Reply:
> the current footer part in the template is probably only text inserted at the end of the invoice and not a "real" footer shown on all pages

correct, I think 🤔

Reply:
so the strange tag name comes from mpdf https://mpdf.github.io/reference/html-control-tags/htmlpagefooter.html
but inside the tag i'm using the footer tag to clarify the use case and put the invoice terms text into a div